UKHomeOffice/nmsw-ui
# Test NMSW template download

Test to verify the UPT responses received are as expected
Test NMSW-1255 : backlink works as expected
Test for NMSW-1293 to verify  supporting file template downloads from voyage report forms
Test for NMSW-1297 download links on FAL5/6 upload pages
Test NMSW-1292 Add wording 'Supernumeraries' to FAL 6 file upload page

## To test
Run with cypress runner
```
npm run cypress:open:dev -- --env MAIL_API_KEY=${MAIL_API_KEY}
```
and choose passenger-status-check.feature
landing-page.feature
file-templates.feature
check-your-answers-page.feature

or in cmd line
```
npm run cypress:run:dev -- --env MAIL_API_KEY=${MAIL_API_KEY} --spec cypress/e2e/features/file-templates.feature

```
```
npm run cypress:run:dev -- --env MAIL_API_KEY=${MAIL_API_KEY} --spec cypress/e2e/features/landing-page.feature
```
```
npm run cypress:run:dev -- --env MAIL_API_KEY=${MAIL_API_KEY} --spec cypress/e2e/features/passenger-status-check.feature
```
```
npm run cypress:run:dev -- --env MAIL_API_KEY=${MAIL_API_KEY} --spec cypress/e2e/features/check-your-answers-page.feature
```
